activesupport/lib/active_support/message_verifier.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require "openssl"
```

Comment:
Let's simplify things here a bit. `MessageEncryptor` also requires `openssl` upfront and I feel like that's OK these days.